pkgs/tools/misc/qjoypad/default.nix

```diff
@@ -32,9 +32,9 @@ stdenv.mkDerivation rec {
       of gaming devices in Linux, and makes the Linux gaming
       experience just a little bit nicer.
     '';
-    homepage = "http://qjoypad.sourceforge.net";
-    license = lib.licenses.gpl2;
-    maintainers = with lib.maintainers; [ astsmtl ];
-    platforms = with lib.platforms; linux;
+    homepage = "https://github.com/panzi/qjoypad";
+    license = licenses.gpl2;
```

Comment:
gpl2Only or gpl2Plus?

Reply:
This project has GPL 2.0 license as I see, so I would assume GPL 2.0 only.